src/lib/Scenes/Artwork/Components/__tests__/AuctionCountDownTimer-tests.tsx

```diff
@@ -1,33 +1,99 @@
 import { Theme, TimeRemaining } from "@artsy/palette"
 import { mount } from "enzyme"
 import { ArtworkFixture } from "lib/__fixtures__/ArtworkFixture"
-import { Settings } from "luxon"
+import moment from "moment"
+import "moment-timezone"
 import React from "react"
-import { AuctionCountDownTimer } from "../AuctionCountDownTimer"
-
-const realNow = Settings.now
-const realDefaultZone = Settings.defaultZoneName
+import { AuctionCountDownTimer, timeUntil } from "../AuctionCountDownTimer"
 
 describe("AuctionCountDownTimer", () => {
-  beforeAll(() => {
-    Settings.defaultZoneName = "America/New_York"
-    Settings.now = () => new Date("2019-08-15T12:00:00+00:00").valueOf()
+  const dateNow = 1565870400000 // 2019-08-15T12:00:00.000Z in milliseconds
+  let oneDayAgo
+  let oneDayFromNow
+  let oneYearFromNow
+
+  beforeEach(() => {
+    jest.useFakeTimers()
+
+    // 2019-08-15T12:00:00.000Z
+    Date.now = () => dateNow
+
+    oneDayFromNow = moment(dateNow)
+      .add(1, "day")
+      .toISOString()
+
+    oneYearFromNow = moment(dateNow)
+      .add(1, "year")
+      .toISOString()
+
+    oneDayAgo = moment(dateNow)
+      .subtract(1, "day")
+      .toISOString()
   })
 
-  afterAll(() => {
-    Settings.now = realNow
-    Settings.defaultZoneName = realDefaultZone
+  describe("timeUntil", () => {
+    it("returns 'Ended' string when auction hasEnded", () => {
+      const startAt = oneDayAgo
+      const liveStartAt = oneDayAgo
+      const endAt = oneDayAgo
+      const auctionState = "hasEnded"
+      const date = timeUntil(startAt, liveStartAt, endAt, auctionState)
+      expect(date).toEqual("Ended Aug 14")
+    })
+
+    it("returns 'Ended' string with year when auction hasEnded over a year ago", () => {
+      const startAt = oneDayAgo
+      const liveStartAt = oneDayAgo
+      const endAt = oneYearFromNow
```

Comment:
These tests are tricky because they both pass in timestamps _and_ refer to `auctionState` (which itself depends on those same timestamps).

In this case, because the auction state is `hasEnded`, this `endAt` timestamp should be sometime in the past.

Reply:
Whoops, meant to put oneYearAgo